.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @iansu @MaxGoo
+* @iansu @@rmagatti
```

Comment:
Isn't Ian gone soon too?

Reply:
Yea, not sure what his plan for his leave is though (I'm getting my access completely revoked, Ian might stay connected). 

Since we don't pay per PR he can change his later if he wants :)